docs/performance.md

```diff
@@ -0,0 +1,74 @@
+# Measuring performance impact
+
+## Summary
+
+The applications being audited are attached an eBPF program, impact on
+performance cannot be ignored. We conducted an experiment to measure
+performance, which this document tries to summarize.
+
+## Testing environment
+
+The experiment was conducted on a typical workstation environment
+running Fedora Linux:
+
+- Hardware: Intel Core i7-1185G7 @ 3.00GHz, 8 core CPU, 32GB memory
+- OS: Fedora 38 with Linux kernel 6.4.11-200.fc38.x86_64
+
+where the following additional components are installed:
```

Comment:
`the following additional components were installed:` instead of `the following additional components are installed:`